tests/attestation/sgxlkl_attests_to_oe/oe_enclave/host/host.cpp

```diff
@@ -0,0 +1,57 @@
+#include <openenclave/host.h>
+#include <stdio.h>
+#include <pthread.h>
+#include <unistd.h>
+#include "tlssrv_u.h"
+
+#define SERVER_PORT "13999"
```

Comment:
Please use a different port to avoid potential conflict.

Reply:
Any suggestion on the port @jxyang ? So I assume the conflict here is because the Huseyin's test, am I understanding correctly?

Reply:
updated to 17500 @jxyang